client/components/domains/transfer-domain-step/transfer-domain-precheck.jsx

```diff
@@ -77,6 +78,10 @@ class TransferDomainPrecheck extends React.Component {
 		this.setState( { currentStep: this.state.currentStep + 1 } );
 	};
 
+	refreshStatus = () => {
+		this.props.refreshStatus( () => this.setState( { unlockCheckClicked: true } ) );
```

Comment:
This creates a function each time you call `refreshStatus`. Make it a member of the class and pass it.

Reply:
Argh. Yep.  Sorry. :-(